### PR TITLE
docs/libraries-and-tools: add etcd-export tool

### PR DESCRIPTION
**Pull request by mickep76**
Added link to Documentation libraries and tools for [etcd-export](https://github.com/mickep76/etcd-export).

### CODE REVIEW

**Comment by yichengq** [comment]
Could you update the commit title to `docs/libraries-and-tools: add etcd-export tool`? This makes commit history much readable.

**Comment by mickep76** [comment]
Should be amended now, thx

**Comment by yichengq** [comment]
@mickep76 You need to squash three commits here and force push it.

File: Documentation/libraries-and-tools.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 - [etcd-lock](https://github.com/datawisesystems/etcd-lock) - Master election & distributed r/w lock implementation using etcd - Supports v2
 - [etcd-console](https://github.com/matishsiao/etcd-console) - A web-base key/value editor for etcd using PHP
 - [etcd-viewer](https://github.com/nikfoundas/etcd-viewer) - An etcd key-value store editor/viewer written in Java
+- [etcd-export](https://github.com/mickep76/etcd-export) - Export/Import Etcd directory as JSON/YAML/TOML and Validate directory using JSON schema
```
Comment (xiang90) [inline Documentation/libraries-and-tools.md]: Etcd -> etcd

**Comment by mickep76** [comment]
Will do new pull req.

**Pull request closed without merging**